src/clj/rems/email/template.clj

```diff
@@ -210,3 +210,20 @@
                               (application-util/get-member-name handler)
                               list
                               (str (:public-url env) "actions"))})))))
+
+(defn reviewer-reminder-email [lang reviewer applications]
+  (with-language lang
+    (fn []
+      (when (not (empty? applications))
+        (let [list (->> applications
+                        (map (fn [application]
+                               (text-format :t.email.reviewer-reminder/application
+                                            (format-application-for-email application)
+                                            (application-util/get-member-name (:application/applicant application)))))
+                        (str/join "\n"))]
+          {:to-user (:userid reviewer)
+           :subject (text :t.email.reviewer-reminder/subject)
```

Comment:
minor: It seems that this new function only differs from the function handler-reminder-email such that :t.email.handler-reminder is changed to :t.email.reviewer-reminder. Maybe it could be given as a parameter to the function so less duplication would be needed?

Reply:
I'm waiting for the third duplicated copy before trying to deduplicate them.